### PR TITLE
[CoW Protocol] Add missing Gnosis Chain solvers

### PR DESCRIPTION
**Pull request by harisang**
This PR adds some missing solvers that are currently operating on Gnosis Chain.

### CODE REVIEW

**Comment by harisang** [comment]
@olgafetisova  Could you double-check that all addresses are correct?

**Review by Hosuke — APPROVED** [review]
LGTM.✅ for syntax.

**Comment by harisang** [comment]
> looks good to me. @harisang let me know if we should wait for @olgafetisova to validate

Pinged @olgafetisova myself so let's wait for her review as well. Thanks for checking with me!

File: models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql
Comment (olgafetisova) [inline models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql:1]: I am also seeing this gnosis solver but did not find it in your PR: 0x67f25791224f0132e6c5102d4b0b3fbf9e8aa69d

File: models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql [hunk 1]
```diff
@@ -49,10 +49,25 @@ known_solver_metadata (address, environment, name) as (
                  (0xd474668a7a9daf34f37c54b22622106277c24166, 'barn', 'MIP'),
                  (0x97d69672e8fe5be64d7d5bbba438ae9b08187667, 'barn', 'MIP'),
                  (0x43e834BE774da64DDe12014e287539E910310255, 'barn', 'Legacy'),
+                 (0x67F25791224f0132E6c5102D4b0B3Fbf9e8Aa69d, 'barn', 'Legacy'),
```
Comment (harisang) [inline models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql:52]: Addressing @olgafetisova 's comment

**Review by olgafetisova — APPROVED** [review]

**Pull request merged**